std/range/package.d

```diff
@@ -4355,15 +4357,372 @@ if (Ranges.length && allSatisfy!(isInputRange, Ranges))
     }
 }
 
+// Simple formatter defined for code generation in ZipShortest to avoid
+// using std.format in CTFE. Only works for very simple patterns
```

Comment:
What's wrong with `format`?

Reply:
I've seen advice against using it in CTFE.

Reply:
It's used in Zip with no issues. I think this overcomplicates the PR.

Reply:
Changed to use `format`.